include/road_network/Road-Model/road.hpp

```diff
@@ -1,72 +1,71 @@
-//
```

Comment:
Deleted the copyright header, @DEVANSH-DVJ

Reply:
Copyright header can be added later (also, there would might be files in which both of us would make changes), also the one before where because Aman was using Xcode.

Reply:
Fine with me, if you both agree to it then I will resolve this conversation.

Reply:
Both of did discuss on this as well as the indentations.